<launch>
    <node pkg="path_planning" exec="trajectory_follower" name="trajectory_follower">
        <param name="odom_topic" value="/odom"/>
        <param name="drive_topic" value="/drive"/>
    </node>
</launch>
